transport_nantes/press/views.py

```diff
@@ -128,7 +128,16 @@
         url = request.POST["url"]
 
     try:
-        response = requests.get(url)
+        # Some websites may filter the default user agent python-requests
+        # We use a custom one to avoid this
+        headers = {
+            # Found this user agent on https://stackoverflow.com/a/57155793/16279937
+            "User-Agent": ("Mozilla/5.0 (Windows NT 10.0; Win64; x64)"
+                           " AppleWebKit/537.36 (KHTML, like Gecko)"
+                           " Chrome/75.0.3770.142"
+                           " Safari/537.36"),
+        }
+        response = requests.get(url, headers=headers)
```

Comment:
This needs a comment in the code explaining why we are ignoring this warning and what measures, if any, we can/should take in light of it.

Reply:
We could use an allow list if we trust say these websites : https://www.acpm.fr/Les-chiffres/Frequentation-Sites-et-Applications/Sites-Grand-Public/Classement-Unifie

I added a comment stating we're trusting links coming there because it's behind a permission only form